app/status/views.py

```diff
@@ -1,9 +1,46 @@
-from flask import jsonify
+from flask import jsonify, current_app
 
 from . import status
 from . import utils
+from ..main.helpers.service import ServiceLoader
+from ..main import main
 
 
 @status.route('/_status')
 def status():
-    return jsonify(status="ok", version=utils.get_version_label())
+
+    # ServiceLoader is the only thing that actually connects to the API
+    service_loader = ServiceLoader(
+        main.config['API_URL'],
+        main.config['API_AUTH_TOKEN']
+    )
+
+    api_response = utils.return_response_from_api_status_call(
+        service_loader.status
+    )
+
+    apis_wot_got_errors = []
+
+    if api_response is None or api_response.status_code is not 200:
```

Comment:
Should be `!= 200`. `is` is an identity comparison, it shouldn't be used for integers/strings.